README.md

```diff
@@ -28,7 +28,9 @@ The endpoints are as follows:
 
 ## Packages
 
-[node-postgres](https://github.com/brianc/node-postgres) is used to interact with the Postgres database when a PowerSync enabled client performs requests to the `/api/data` endpoint.
+[node-postgres](https://github.com/brianc/node-postgres) is used to interact with the Postgres database when a Postgres + PowerSync enabled client performs requests to the `/api/data` endpoint.
```

Comment:
```suggestion
[node-postgres](https://github.com/brianc/node-postgres) is used to interact with the Postgres database when a client performs requests to the `/api/data` endpoint.
```

The client itself is not specific to MongoDB or Postgres, so I'd call it just a "client" or "PowerSync-enabled client"

Reply:
Thanks! This one was very silly.